Shuttle-back/src/main/java/com/shuttle/driver/Driver.java

```diff
@@ -4,4 +4,14 @@
 
 public class Driver extends User {
 
+    public Driver() {
+    }
+
+    public Driver(Long id, String name, String surname, String profilePicture, String telephoneNumber, String email, String address, String password) {
+        super(id, name, surname, profilePicture, telephoneNumber, email, address, password);
+    }
+
+    public DriverDTO parse2DTO() {
```

Comment:
Domain class shouldn't know about the DTO, because multiple DTO types could be constructed from the same domain class. Use ```DriverDTO::DriverDTO(Driver)``` instead.